### PR TITLE
update docs for istioctl proxy-config cmd

### PR DESCRIPTION
**Pull request by DreamRivulet**
Update docs for `istioctl proxy-config` cmd as this cmd is changed since release 1.0.

### CODE REVIEW

**Comment by DreamRivulet** [comment]
@ayj , could you please help to take a review? Thanks in advance.

File: content/help/faq/telemetry/life-of-a-request.md [hunk 1]
```diff
@@ -7,11 +7,20 @@ You can enable [tracing](/docs/tasks/telemetry/distributed-tracing/) to figure o
...
+    $ istioctl proxy-config bootstrap productpage-v1-bb8d5cbc7-k7qbm
+
+    # Retrieve information about cluster configuration for the Envoy instance in the specified pod.
+    $ istioctl proxy-config cluster productpage-v1-bb8d5cbc7-k7qbm
+
+    # Retrieve information about listener configuration for the Envoy instance in the specified pod.
+    $ istioctl proxy-config listener productpage-v1-bb8d5cbc7-k7qbm
+
+    # Retrieve information about route configuration for the Envoy instance in the specified pod.
+    $ istioctl proxy-config route productpage-v1-bb8d5cbc7-k7qbm
```
Comment (liamawhite) [inline content/help/faq/telemetry/life-of-a-request.md]: `istioctl proxy-config endpoints productpage-v1-bb8d5cbc7-k7qbm` was also recently added.

File: content/help/ops/component-debugging/index.md [hunk 2]
```diff
@@ -10,24 +10,42 @@ how to get under the hood.
...
-For example, to retrieve the configured clusters in an Envoy via the admin interface run the following command:
+You can get an overview of your mesh via `proxy-status` command:
 
 {{< text bash >}}
-$ istioctl proxy-config endpoint <pod-name> clusters
+$ istioctl proxy-status
 {{< /text >}}
 
-To retrieve endpoints for a given pod in the application namespace from Pilot, run the following command:
+If a proxy is missing from the output list it means that it is not currently connected to a Pilot instance so will not be receiving any configuration.
```
Comment (liamawhite) [inline content/help/ops/component-debugging/index.md]: Also if it is stale that likely means there are networking issue or Pilot needs to be scaled.

File: content/help/ops/component-debugging/index.md [hunk 3]
```diff
@@ -10,24 +10,42 @@ how to get under the hood.
...
-$ istioctl proxy-config pilot -n application <pod-name> eds
+$ istioctl proxy-config listener <pod-name> [flags]
 {{< /text >}}
 
-The `proxy-config` command also allows you to retrieve the state of the entire mesh from Pilot using the following command:
+To retrieve information about route configuration for the Envoy instance in the specified pod, run the following command:
 
 {{< text bash >}}
-$ istioctl proxy-config pilot mesh ads
+$ istioctl proxy-config route <pod-name> [flags]
```
Comment (liamawhite) [inline content/help/ops/component-debugging/index.md]: Also endpoints, as above.

**Pull request closed without merging**